### PR TITLE
Remapping Pool Types;

### PR DESCRIPTION
**Pull request by lgahdl**
Remapping PoolTypes from API, like PHANTOM_STABLE, to SDK PoolType, like PoolType.ComposableStable

### CODE REVIEW

**Review by brunoguerios — CHANGES_REQUESTED** [review]
The PR is picking up way more diff than it should. 
Maybe the base branch is incorrect?
Please clean it up and re-request the review.

**Comment by lgahdl** [comment]
> The PR is picking up way more diff than it should. Maybe the base branch is incorrect? Please clean it up and re-request the review.

I merged the main and composable stable init branches last changes into it, now I think it's cleaned up

File: src/entities/createPool/composableStable/createPoolComposableStable.ts [hunk 1]
```diff
@@ -46,9 +46,9 @@ export class CreatePoolComposableStable implements CreatePoolBase {
             input.name || input.symbol,
             input.symbol,
             tokens,
-            input.amplificationParameter,
+            BigInt(input.amplificationParameter),
```
Comment (brunoguerios) [inline src/entities/createPool/composableStable/createPoolComposableStable.ts:49]: Why don't we expect the user to provide the proper types? Seems to make more sense than casting it here.

File: src/entities/removeLiquidity/removeLiquidity.ts [hunk 2]
```diff
@@ -22,12 +23,11 @@ export class RemoveLiquidity {
...
-            GYRO3: new RemoveLiquidityWeighted(),
-            GYROE: new RemoveLiquidityWeighted(),
-            WEIGHTED: new RemoveLiquidityWeighted(),
-            // PHANTOM_STABLE === ComposableStables in API
-            PHANTOM_STABLE: new RemoveLiquidityComposableStable(),
+            [PoolType.Gyro2]: new RemoveLiquidityWeighted(),
+            [PoolType.Gyro3]: new RemoveLiquidityWeighted(),
+            [PoolType.GyroE]: new RemoveLiquidityWeighted(),
+            [PoolType.Weighted]: new RemoveLiquidityWeighted(),
+            [PoolType.ComposableStable]: new RemoveLiquidityComposableStable(),
```
Comment (brunoguerios) [inline src/entities/removeLiquidity/removeLiquidity.ts:30]: Nice solution! We use poolTypes while letting the user provide custom types as string 👏

**Review by brunoguerios — APPROVED** [review]
LGTM!

File: src/entities/removeLiquidity/removeLiquidity.ts (hunk 2 above)
Comment (lgahdl) [reply src/entities/removeLiquidity/removeLiquidity.ts:30]: Thanks! 😊

**Pull request merged**